### ISSUE TITLE
Large number of people using extensions (feature request/suggestion) 

### ISSUE THREAD

**Issue by mediocreatmybest** [TheLastBen/fast-stable-diffusion#1401]
Hi, 

Due to the large number of people using additional extensions with the automatic1111 colab, it might be worth creating a known working / known issues extension file, similar to the readme.md.

Having a list of problem extensions and known ones that don't have issues might go a long way in cutting down on the issue list.
Not everything is fixable and might make it quicker in closing lingering issues. 

Thanks!

**Comment by TheLastBen** [comment]
Except the dreambooth extension which requires gradio 3.15 for some reason, all the extension should work correctly.

**Comment by mediocreatmybest** [comment]
Thanks, might be something the community or contributors would be willing to help with? 
Food for thought. 

The following ones I've found issues with.  

- https://github.com/zero01101/openOutpaint-webUI-extension --> Could be due to its use of API? I'm only guessing.
- https://github.com/d8ahazard/sd_smartprocess --> Previously installing this would break the startup of WebUI in Colab.
- https://github.com/d8ahazard/sd_dreambooth_extension --> Quite strict requirements listed that can break items. It does work with Gradio 3.9.0 though (from what I can tell)

I haven't gone through all of the ones available in the interface just a small handful. 

Also, I'm not suggesting changing the structure of how the colab is built, I did a quick test for myself to try see what versions within pip seem to not throw up errors after everything is installed and allow the WebUI to start, more just an FYI for anyone interested, that includes updated accelerate, transformers, latest xformers binary as well. 

https://github.com/mediocreatmybest/gaslightingeveryone/blob/main/Colab/files/test_requirements.txt
